### ISSUE TITLE
`look` does not work after game restarts.

### ISSUE THREAD

**Issue by flosincapite** [Cronenbrogues/cronenbroguelike#69]
After restarting due to death, the initial room description will be repeated twice; subsequent calls to `look` will then fail to do anything. This does not happen if the player dies by suicide or by cheating. This only seems to occur when the player is killed by a fish man. Reproduce by allowing yourself to be killed, e.g.

```
> look

Fish man hits you for 6 damage!

Health damaged by 6.

You die.

...

You recall your death by the fins of a fish man. The memory fades away.

You know only that you have been here for interminable years, that you have died innumerable times, and that someone once told you there was a way out. You were told this an eon ago, or maybe a day, but
the stubborn hope of escape glisters in your mind.

A desk is strewn with sheaves of paper. Little of sense is written there.

A smoker stands meditatively, cigarette in hand. He looks cool, maybe cooler than you.

Exits are through a murky tunnel.

A desk is strewn with sheaves of paper. Little of sense is written there.

A smoker stands meditatively, cigarette in hand. He looks cool, maybe cooler than you.

Exits are through a murky tunnel.

> look

> 
```

**Comment by abjose** [comment]
Here's a stacktrace from the first call to `look()`:
``` python
File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
  "__main__", mod_spec)
File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
  exec(code, run_globals)
File "/cronenbroguelike/cronenbroguelike/__main__.py", line 24, in <module>
  game.main()
File "/cronenbroguelike/cronenbroguelike/game.py", line 110, in main
  adventurelib.start()
File "/home/BOSDYN/ajose/.local/lib/python3.6/site-packages/adventurelib.py", line 627, in start
  _handle_command(cmd)
File "/home/BOSDYN/ajose/.local/lib/python3.6/site-packages/adventurelib.py", line 600, in _handle_command
  func(**args)
File "<string>", line 4, in wrapped
File "/cronenbroguelike/cronenbroguelike/commands.py", line 70, in look
  _resolve_attack(character, action.attack)
File "/cronenbroguelike/cronenbroguelike/commands.py", line 136, in _resolve_attack
  -1 * damage, cause=f"the fins of {util.a(attacker.name)}")
File "/cronenbroguelike/engine/actor.py", line 98, in heal_or_harm
  self._owner.die(cause=self.last_cause)
File "/cronenbroguelike/engine/actor.py", line 243, in die
  self._death_throes(self)
File "/cronenbroguelike/cronenbroguelike/game.py", line 99, in _restart
  commands.enter_room(_G.player.current_room)
File "/cronenbroguelike/cronenbroguelike/commands.py", line 17, in enter_room
  _look()
File "/cronenbroguelike/cronenbroguelike/commands.py", line 25, in _look
  traceback.print_stack()
```

And from the 2nd:
``` python
File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
  "__main__", mod_spec)
File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
  exec(code, run_globals)
File "/cronenbroguelike/cronenbroguelike/__main__.py", line 24, in <module>
  game.main()
File "/cronenbroguelike/cronenbroguelike/game.py", line 110, in main
  adventurelib.start()
File "/home/BOSDYN/ajose/.local/lib/python3.6/site-packages/adventurelib.py", line 627, in start
  _handle_command(cmd)
File "/home/BOSDYN/ajose/.local/lib/python3.6/site-packages/adventurelib.py", line 600, in _handle_command
  func(**args)
File "<string>", line 4, in wrapped
File "/cronenbroguelike/cronenbroguelike/commands.py", line 74, in look
  _look()
File "/cronenbroguelike/cronenbroguelike/commands.py", line 25, in _look
  traceback.print_stack()
```
What I think is happening:
1. give command to `look`
2. in `look()`, we generate NPC actions which in this case kills the PC
3. the PC's `_death_throes()` callback runs, which restarts the game and calls `look()` via `enter_room()` (also clearing the `just_died` flag)
4. after all of this, we finish processing the original `look` command by calling `_look()`

So it ends up printing twice. I haven't been able to reproduce the part where `look` doesn't work after seeing it printed twice.

**Comment by flosincapite** [comment]
Fixed by #76
